scopes/defender/jest/jest.worker.ts

```diff
@@ -35,6 +39,10 @@ export class JestWorker {
             `${__dirname}/watch.js`,
             {
               specFiles: testFiles,
+              shouldRunTestSuite: async (specFile: string) => {
```

Comment:
naming is confusing, you have `shouldRunTestSuite` which return a boolean. and you have something with the same name which is function that return a boolean
change one of the names